device/ipv4.yaml

```diff
@@ -57,7 +59,23 @@ components:
           type: string
           format: ipv4
           default: 0.0.0.0
-        
+
+    Device.Ipv4ManualGatewayMAC:
+      description: >- 
+        Object to set Manual Gateway MAC
+      type: object
+      properties:
+        enable:
+          description: >-
+            By default Manual Gateway MAC is not enabled and the resolved Gateway is used.
+            To overwrite the resolved Gateway with Manual Gateway(address), set enable to True.
```

Comment:
Maybe something like :
By default the destination MAC used for control and data packets sent from this IPv4 endpoint is learnt by sending ARP Requests ( or ND for IPv6 ) to the configured gateway IP . 
Enabling this would mean that ARP ( or ND for IPv6 resolution ) will not be used for learning MAC of  connected device.  This is needed if ARP ( or ND for IPv6) resolution is not possible in some scenario with the device where the gateway is configured.

Reply:
All of this description should be added to the container object description.

Reply:
updated